src/main/java/codeu/controller/ConversationServlet.java

```diff
@@ -112,39 +112,25 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     conversationTitle = conversationTitle.trim();
 
     if (conversationTitle.equals("")) {
-      request.setAttribute("error", "Please enter at least one letter or number");
+      request.setAttribute("error", "Please enter at least one character");
       request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
       return;
     }
 
-    if (!conversationTitle.matches("[\\w*\\s*]*")) {
-      request.setAttribute("error", "Please enter only letters and numbers.");
-      request.getRequestDispatcher("/WEB-INF/view/conversations.jsp").forward(request, response);
-      return;
-    }
-
-    String conversationTitleStored = conversationTitle.replaceAll(" ", "-");
-
-    if (conversationStore.isTitleTaken(conversationTitleStored)) {
```

Comment:
Are we using this method anywhere else? If not, let's remove it from `ConversationStore`?